ConsoleApp1/ConsoleApp1/ChatBot.cs

```diff
@@ -0,0 +1,41 @@
+using System.Globalization;
+
+namespace ConsoleApp1;
+
+public class ChatBot
+{
+    public static void Chatting()
+    {
+        Console.WriteLine("Здравствуйте! Введите вашу Фамилию.");
+        var lastName = Console.ReadLine();
+
+        while (lastName != null && (char.IsLower(lastName[0]) || lastName.Any(char.IsDigit)))
+        {
+            Console.WriteLine("Введен неверный формат данных, введите вашу Фамилию повторно");
+            lastName = Console.ReadLine();
+        }
+
+        Console.WriteLine("Введите ваше Имя.");
+        var name = Console.ReadLine();
+        while (name != null && (char.IsLower(name[0]) || name.Any(char.IsDigit)))
+        {
+            Console.WriteLine("Введен неверный формат данных, введите ваше Имя повторно");
+            name = Console.ReadLine();
+        }
```

Comment:
It does not fit the task. Need a transformation, not a validation. Read the task